numpy/core/tests/test_multiarray.py

```diff
@@ -6355,6 +6355,7 @@ def test_string(self):
         assert_equal(np.where(False, b, a), "abcd")
 
 
+@np.testing.dec.skipif(IS_PYPY, "no getsizeof() in PyPy")
```

Comment:
They don't, need to use a plain old `if` statement around the class declaration.